.golangci.yml

```diff
@@ -0,0 +1,14 @@
+linters:
+  disable:
+    - deadcode
+    - errcheck
+    - gosimple
+    #- govet
+    - ineffassign
+    - staticcheck
+    #- structcheck
+    #- typecheck
```

Comment:
Why do these need to be commented out? Can't they just  be removed?

Reply:
Good point.